hazelcast/src/test/java/com/hazelcast/config/YamlConfigSchemaValidatorTest.java

```diff
@@ -81,4 +81,9 @@ public void lenientPrimitiveValidationIsEnabled() {
     public void emptyObject() {
         new YamlConfigBuilder(getClass().getResourceAsStream("/com/hazelcast/config/empty-object.yaml")).build();
     }
+
+    @Test(expected = InvalidConfigurationException.class)
+    public void witoutRootObject() {
```

Comment:
```suggestion
    public void withoutRootObject() {
```
Please change this as well. Then, we should be good to go.

Reply:
fixed.
thanks for patience

Reply:
Thanks, no problem. Thank you too.